BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -154,19 +155,25 @@
 
     <!-- Note -->
     <string name="note">Notiz</string>
+
+    <!-- Note List -->
+    <string name="empty_notes_list">Derzeit sind keine Notizen vorhanden.</string>
```

Comment:
```suggestion
```

I think we don't need this and should use "empty_list" instead, which is standard throughout the app.

Reply:
You are right - this is done.